sources/packages/web/src/components/aest/InstitutionUserSummary.vue

```diff
@@ -1,131 +1,126 @@
 <template>
-  <v-container>
-    <div class="row">
-      <div class="col align-self-start">
-        <span class="category-header-large color-blue">
-          All Users({{ usersListAndCount.totalUsers }})
-        </span>
-      </div>
-      <div>
-        <InputText
-          v-model="searchBox"
-          placeholder="Search User"
-          @keyup.enter="searchUserTable()"
-        />
-      </div>
-      <div>
-        <v-btn @click="searchUserTable()" color="primary" tile class="ml-2">
-          <font-awesome-icon icon="search" />
-        </v-btn>
-      </div>
-      <div v-if="clientType === ClientIdType.Institution" class="mx-2">
-        <v-btn class="float-right" @click="openNewUserModal()" color="primary">
-          <v-icon size="25" left> mdi-plus-circle </v-icon>
-          Add New User
-        </v-btn>
+  <!-- This component is shared between ministry and student users -->
+  <div class="row">
+    <div class="col align-self-start">
+      <span class="category-header-large color-blue">
+        All Users({{ usersListAndCount.totalUsers }})
+      </span>
+    </div>
+    <div>
+      <InputText
+        v-model="searchBox"
+        placeholder="Search User"
+        @keyup.enter="searchUserTable()"
+      />
+    </div>
+    <div>
+      <v-btn @click="searchUserTable()" color="primary" tile class="ml-2">
+        <font-awesome-icon icon="search" />
+      </v-btn>
+    </div>
+    <div v-if="clientType === ClientIdType.Institution" class="mx-2">
+      <v-btn class="float-right" @click="openNewUserModal()" color="primary">
```

Comment:
Until we finalize the approach on color and background to buttons, can you use the class primary-btn-background? it will make the button to appear in figma's blue color.